services/ui-src/src/utils/other/export.tsx

```diff
@@ -48,15 +48,21 @@ export const renderDrawerDataCell = (
     const fieldResponseData = entity[formField.id];
     return (
       <Box key={entity.id + formField.id} sx={sx.entityBox}>
-        <Text sx={sx.entityName}>{entity.name}</Text>
-        {renderResponseData(
-          formField,
-          fieldResponseData,
-          entityResponseData,
-          pageType,
-          entityIndex,
-          notApplicable
-        )}
+        <ul>
```

Comment:
Info: There's a Chakra way to do this too! See [Chakra List here](https://chakra-ui.com/docs/components/list). If we did it this way we could use `sx` prop for styling and I think also pass `"none"` as the `styleType`